### ISSUE TITLE
Symbol ao_file_extension not found

### ISSUE THREAD

**Issue by rokf** [TheLinx/lao#9]
I'm trying to run `examples/ao_live.lua` which crashes with the message

```
lua: error loading module 'ao' from file '/usr/local/lib/lua/5.1/ao.so':
  Error relocating /usr/local/lib/lua/5.1/ao.so: ao_file_extension: symbol not found
stack traceback:
  [C]: ?
  [C]: in function 'require'
  examples/ao_live.lua:1: in main chunk
  [C]: ?
```
I'm using **Alpine Linux** with `libao-1.2.0-r1`. Any idea what could be wrong since the version is the most recent one (1.2)?

**Comment by daurnimator** [comment]
I found this bug: https://trac.xiph.org/ticket/1841 but that says it should be fixed in 1.2.0
Can you run `nm -D -C --defined-only /usr/lib/libao.so`?
What I get:

```
0000000000003860 T ao_append_global_option
00000000000037b0 T ao_append_option
0000000000003ea0 T ao_close
0000000000003f90 T ao_default_driver_id
0000000000003f30 T ao_driver_id
00000000000041c0 T ao_driver_info
0000000000004220 T ao_driver_info_list
0000000000004230 T ao_file_extension
0000000000003880 T ao_free_options
0000000000002db0 T ao_initialize
0000000000004290 T ao_is_big_endian
00000000000050a0 T ao_open_file
0000000000005090 T ao_open_live
00000000000039a0 T ao_play
00000000000038c0 T ao_shutdown
0000000000000000 A LIBAO4_1.1.0
0000000000000000 A LIBAO4_1.2.0
```

If not there then i think file a bug against libao on the alpine tracker?

**Comment by rokf** [comment]
Yup, it's missing. Getting:
```
0000000000000000 A LIBAO4_1.1.0
000000000000302d T ao_append_global_option
0000000000002fa6 T ao_append_option
0000000000003479 T ao_close
0000000000003544 T ao_default_driver_id
0000000000003506 T ao_driver_id
00000000000036cd T ao_driver_info
00000000000036ee T ao_driver_info_list
000000000000303f T ao_free_options
00000000000027f0 T ao_initialize
000000000000371f T ao_is_big_endian
00000000000041c0 T ao_open_file
00000000000041b9 T ao_open_live
0000000000003125 T ao_play
0000000000003073 T ao_shutdown
```
Thanks! Will report the issue.
